0006/README.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 0006
+title: Resiliency in Tinkerbell Provisioner.
+status: Ideation
+authors: Chitrabasu Khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
+This RFD proposes to have a resilient framework for Tinkerbell Provisioner.
+
+
+## Goals:
+
+- Bring resiliency in Tinkerbell. 
+- Tinkerbell as a setup should be highly available.
+
+## Content: 
+
+- We would like to have a framework where Tinkerbell as a solution is fault-tolerant.
```

Comment:
Can you add a couple 5-6 different scenarios that you would like to fix in this proposal?

Reply:
done

Reply:
will push change in some time.

Reply:
done